net/netfilter/nf_tables_api.c

```diff
@@ -5646,7 +5666,7 @@ static int nft_add_set_elem(struct nft_ctx *ctx, struct nft_set *set,
 	return 0;
 
 err_set_full:
-	set->ops->remove(ctx->net, set, &elem);
+	nft_setelem_remove(ctx->net, set, &elem);
```

Comment:
This is technically introduced in 
`netfilter: nftables: add catch-all set element support` 
https://github.com/ctrliq/kernel-src-tree/pull/2/commits/628cf6169148cf5453ff84705ea254069b509428

The upstream is different but the code here matches the `4.18.0-534` we've based everything off of:
https://github.com/torvalds/linux/commit/aaa31047a6d25da0fa101da1ed544e1247949b40#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR5947

Reply:
Fixed in the next patch series.